src/Bootstrap.php

```diff
@@ -199,4 +209,22 @@ public function addDaenerysCommands(Application $application)
             }
         }
     }
+
+    /**
+     * Runs the code to extend models.
+     */
+    public function extendModels()
+    {
+        AnnotationRegistry::registerLoader("class_exists");
+
+        $modelExtender = new ModelExtender();
+
+        foreach ($this->libraryConfigurationManager->getConfigurations() as $config) {
+            $modelExtensions = $config->getSubKeyIfItExists(["modelExtensions"]);
+
+            if ($modelExtensions) {
+                $modelExtender->addMore($modelExtensions);
```

Comment:
Can there be errors here?

Reply:
No, getSubKeyIfItExists returns null if the key does not exist, so the rest of the method gets silently ignored. If there are any errors (like, class does not exist), the module tests should report on it.